src/storage/noteStorageAPI.js

```diff
@@ -442,3 +443,28 @@ function remove(note) {
     spinner.hide();
   });
 }
+
+function move({ noteId, newParentId, }) {
+  spinner.show();
+  const request = gapi.client.request({ 
+    path: '/drive/v2/files/' + noteId, 
+    method: 'PUT', 
+    params: { 'uploadType': 'multipart', 'alt': 'json' }, 
+    body: { parents: [{ 'id': newParentId }] }
+  });
+
+  return new Promise((resolve, reject) => {
+    request.execute(function (response, responseRaw) {
+      toast.error('tree re-rendering is not implemented yet');
```

Comment:
`noteStorageAPI` is not responsible for re-rendering the tree.
It only provides CRUD for the notes remote storage.

`toast.error()` should be called from the saga. Saga is the place where we have the logic for what to do when the http request for moving note has succeeded or failed.